crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -76,17 +78,60 @@ impl DynamicScene {
         scene
     }
 
+    /// Create a new dynamic scene from a given [`World`] at a given [`Entity`].
+    ///
+    /// All descendants entities from the given one will be extracted.
+    pub fn from_world_at_root(
+        world: &World,
+        root: Entity,
```

Comment:
Personally, I think this method would have limited practical utility. Who wants to create a scene from *just one* entity (+ children)? Sure, sometimes you might have such a use case, but it seems needlessly limited. What if you want a second entity? Or a few more?

Why not change this to `impl IntoIterator<Item=Entity>`, allowing users to provide any number of root entities?

Would add a little extra complexity to the body of the function. Perhaps there should be some sort of deduplication logic to prevent redundant/duplicate entities in the scene. One suggestion might be to use an intermediary `HashSet` instead of a `Vec`. Or maybe we could just not do any deduplication (just accumulate all the entities into the scene without regard for duplicates), and document it as a caveat / leave it to the users to make sure they don't have duplicate entities in their list of roots.

It would make this function much more practical and useful.

Reply:
Would you prefer to be able to append to an existing scene? Or just creating from multiple entities

Reply:
Well now that you have said it ... i guess both? :laughing: